tests/test_dynamo/test_jobs.py

```diff
@@ -437,6 +437,7 @@ def test_update_job(tables):
             'user_id': 'user1',
             'status_code': 'status2',
             'request_time': '2000-01-01T00:00:00+00:00',
+            'processing_time_in_seconds': Decimal('1.23'),
```

Comment:
Are we concerned about the fact that using `Decimal` results in capturing the entire float value in the database, i.e. `Decimal(1.23)` is equivalent to `Decimal('1.229999999999999982236431605997495353221893310546875')`? Would it be better to ensure that the "actual" decimal values get put in the database?

Reply:
Ah, my mistake. I did not notice that the float is converted to a string first.